src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    static final int SUPPLY_BUY_INDEX = 0;
+    static final int AMOUNT_INDEX = 1;
+    static final String SPLIT_REGEX = ",";
+    private int supply = 0;
+    private int buy = 0;
+    private int result;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readHealper(fromFileName);
+
+        result = supply - buy;
+
+        writeHelper(toFileName);
+    }
+
+    public void readHealper(String fromFileName) {
```

Comment:
helper methods should be private
```suggestion
    private void readFromFile(String fromFileName) {
```